mesonbuild/envconfig.py

```diff
@@ -313,7 +313,7 @@ def is_darwin(self) -> bool:
         """
         Machine is Darwin (iOS/tvOS/OS X)?
         """
-        return self.system in {'darwin', 'ios', 'tvos'}
+        return self.system == 'darwin'
```

Comment:
This will probably break existing use cases in the wild. It also looks like it conflicts with changes you made elsewhere? What is the purpose of the change -- the commit message doesn't have much in the way of "why"s and definitely doesn't mention this change at all.

Reply:
This was first added in f59786e81d82ef061dab19cf4ce8981943bab9c3. I suppose people are supposed to be able to both use system='ios' and system='darwin' subsystem='ios'? I'll revert this then.